.github/workflows/ow_k8s.yml

```diff
@@ -77,10 +77,11 @@ jobs:
     - name: Checkout repo with Helm values
       uses: actions/checkout@v2
       with:
-        path: wlan-testing
+        repository: Telecominfraproject/wlan-cloud-ucentral-deploy
+        path: wlan-cloud-ucentral-deploy
```

Comment:
We should probably also checkout the target branch to get deployment related to the changes for the specific version

Reply:
@dunaev-opsfleet yeah missed that, thanks!